### DISCUSSION TITLE
Components inside tab container (plugin bb-plugin-TabContainer) became invisible

### DISCUSSION THREAD

**Discussion by E-JOKE** [Budibase/budibase#10875] [Help] [answered]
Looks like Tab container is not working anymore. All components inside the tab container are still visible on the left pane in Budibase design mode, but are invisible in visual preview. Tested this in an existing project and a new project. Both fail. Previous version of Budibase: same problem. Any ideas?
[Budibase_Tab.pdf](https://github.com/Budibase/budibase/files/11727183/Budibase_Tab.pdf)

**Comment by melohagan** [comment]
Hey @E-JOKE 

I'm not seeing any issues with the Tabs components. Tested there now on **v2.7.14**. 

Is it possible that it is related to this issue? https://github.com/Budibase/budibase/issues/9260

You could try adding another Tab component to the screen to see if it works, or try deleting and re-adding the Tabs components in the plugins page.

**Comment by E-JOKE** [comment, marked as answer]
Hi melohagan,

 

It miraculously started working again this morning.

 

Greetings,

Erik

 

Van: melohagan ***@***.***> 
Verzonden: dinsdag 13 juni 2023 15:21
Aan: Budibase/budibase ***@***.***>
CC: E-JOKE ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [Budibase/budibase] Components inside tab container (plugin bb-plugin-TabContainer) became invisible (Discussion #10875)

 

Hey @E-JOKE <https://github.com/E-JOKE> 

I'm not seeing any issues with the Tabs components. Tested there now on v2.7.14.

Is it possible that it is related to this issue? #9260 <https://github.com/Budibase/budibase/issues/9260> 

You could try adding another Tab component to the screen to see if it works, or try deleting and re-adding the Tabs components in the plugins page.
